src/schema/generated-schema-types.ts

```diff
@@ -4007,7 +4008,7 @@ export type SignatureContentInput = {
 };
 
 export type SignatureInput = {
-  content?: InputMaybe<SignatureContentInput>;
+  content?: InputMaybe<Array<InputMaybe<SignatureContentInput>>>;
```

Comment:
This line is not relevant to this functionality however its getting generated after building the code hence I have kept it in the PR.